File: sample/src/main/java/vn/tiki/sample/util/PasswordValidator.java
package vn.tiki.sample.util;

public class PasswordValidator {

  /**
   * Validate password
   *
   * @param password password for validation
   * @return true valid password, false invalid password
   */
  public boolean validate(final String password) {
    return password != null && password.length() >= 6;
  }
}

File: sample/src/main/java/vn/tiki/sample/util/PasswordValidator.kt
package vn.tiki.sample.util

class PasswordValidator {

  /**
   * Validate password
   *
   * @param password password for validation
   * @return true valid password, false invalid password
   */
  fun validate(password: String?): Boolean {
    return password != null && password.length >= 6
  }
}
